src/parser/druid/balance/modules/talents/azeritetraits/StreakingStars.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import Events from 'parser/core/Events';
+import DAMAGING_ABILITIES from 'parser/druid/balance/constants';
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import { calculateAzeriteEffects } from 'common/stats';
+import AzeritePowerStatistic from 'interface/statistics/AzeritePowerStatistic';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+
+const streakingStarsStats = traits => Object.values(traits).reduce((obj, rank) => {
+  const [damage] = calculateAzeriteEffects(SPELLS.STREAKING_STARS.id, rank);
+  obj.damage += damage;
+  return obj;
+}, {
+  damage: 0,
+});
+
+/**
+ * Streaking Stars,  Azerite Power
+ * Requires Druid (Balance)
+ * While Celestial Alignment is active, your damaging spells call a Streaking Star,
+ * dealing an additional 3876 damage when they are not a repeat of the previous ability.
+ * Example log: https://www.warcraftlogs.com/reports/ma43rhYpFzAQHxtv#fight=21&type=damage-done&source=27&ability=272873
+ *
+ * Streaking Stars will never check spells outside of Celestial Alignment window.
+ * You can cast Lunar Strike > Celestial Alignment > Lunar Strike and still get
+ * the Streaking Stars proc on the second Lunar Strike. Because of this we don't
+ * need to keep track of casts outside of Celestial Alignment windows.
+ * Example: https://www.warcraftlogs.com/reports/jnAYqfr74FQRhxZG#fight=16&type=casts&source=8&start=4692350&end=4714505&pins=0%24Separate%24%23244F4B%24auras-gained%240%240.0.0.Any%24140177772.0.0.Druid%24true%24140177772.0.0.Druid%24false%24102560%5E0%24Separate%24%23909049%24damage%240%240.0.0.Any%24140177772.0.0.Druid%24true%240.0.0.Any%24false%24194153%5E0%24Separate%24%23a04D8A%24damage%240%240.0.0.Any%24140177772.0.0.Druid%24true%240.0.0.Any%24false%24272873&ability=194153
+ */
+class StreakingStars extends Analyzer {
+
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+  };
+
+  buffToTrack = SPELLS.CELESTIAL_ALIGNMENT;
+  celestialAlignmentRunning = false;
+  lastCastSpellId = 0;
+
+  totalCastsDuringCelestialAlignment = 0;
+  badCasts = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.STREAKING_STARS.id);
+    if (!this.active) {
+      return;
+    }
+    if (this.selectedCombatant.hasTalent(SPELLS.INCARNATION_CHOSEN_OF_ELUNE_TALENT.id)) {
+      this.buffToTrack = SPELLS.INCARNATION_CHOSEN_OF_ELUNE_TALENT;
+    }
+
+    const { damage } = streakingStarsStats(this.selectedCombatant.traitsBySpellId[SPELLS.STREAKING_STARS.id]);
```

Comment:
There's no need for this function since streaking stars is its own damage source. If you check the logs the dmg numbers doesn't match since you don't take into account any damage mods like crit, vers, CA/INC, any dmg taken reductions/increases on the target, etc. 

You can just use the DamageDone getter you made instead, that should give you the correct numbers.